lib/api.js

```diff
@@ -616,3 +616,23 @@ exports.reorder_metadata_fields = function reorder_metadata_fields(order_by, dir
   const params = { order_by, direction };
   return call_api("put", ["metadata_fields", "order"], params, callback, options);
 };
+
```

Comment:
Maybe add doc strings to functions?

Reply:
Usage of doc strings is not consistent across the sdk, so I haven't added it on purpose.

I'd like to even argue they should be removed completely, because they tend to not be updated when the code changes and may be a cause for confusion if that's the case.

Reply:
Ideally we would like to generate a reference, like in JS SDK:
https://cloudinary.com/documentation/sdks/js/url-gen/index.html
The reason it is not consistent, is that old code did not have it. All the new code is well documented. (at least we try)